home.css

```diff
@@ -35,7 +35,7 @@
 
 .grid-4-col{
     display: grid;
-    grid-template-columns:repeat(auto-fill, minmax(20rem, 1fr));
+    grid-template-columns:repeat(auto-fit, minmax(20rem, 1fr));
     place-content: center ;
     column-gap: 1rem;
     row-gap: 1rem;
```

Comment:
you can use the shorthand property for row gap and column gap

Reply:
will keep that in mind.